src/gui/systray.cpp

```diff
@@ -160,6 +136,40 @@ void Systray::create()
     }
 }
 
+void Systray::setupContextMenu()
+{
+    _contextMenu = new QMenu();
```

Comment:
once `_contextMenu` has been already set to a valid pointer, executing this line will result in a memory leak if I understand the behavior of `QPointer` correctly.

what you want is either `std::auto_ptr` or `QScopedPointer`

Reply:
Both of these alternative wrappers cause an immediate segfault on application start

Reply:
I have added a quick `_contextMenu.clear()` before the assignment which should resolve the memory leak issue?

Reply:
Original QMenu now gets deallocated without any explosions